website/blog/2023-12-08-silver.md

```diff
@@ -0,0 +1,585 @@
+
+Metals tree view enables users to navigate their workspace in a more structured
+way by directly showing the build modules and their packages, classes, objects
+etc. It would also allow users to navigate their dependency jars. Previously,
+this was based on special Scala 2 signature information, which means it wouldn't
+work in Scala 3.
+
+From this release we changed it to use the same information that is used for
+indexing in case of jars and semanticdb for local workspace files. This means
+that Scala 3 is fully supported and this has been the last big feature missing
+for full parity between Scala 2 and 3.
+
+![tree-view](https://i.imgur.com/Vun4kJh.gif)
+
+All the existing features should now work for Scala 3 as well. This includes the
+reveal command which allows to quickly reveal the current file in the tree view
+including inside jars.
+
+![tree-view-reveal](https://i.imgur.com/xW0W3I3.gif) Additionally, we improved
+icons used by the tree view. They will now use the default VS Code icons for the
+most part, which should make them more consistent with the rest of the editor.
+
+## Go to definition for scaladoc
+
+When writing scaladoc for your methods or classes you can use links, which are
+of a form `[[name.name.ClassName]]`. Thanks to
+[kasiaMarek](https://github.com/kasiaMarek) Metals will now allow you to
+navigate to the target of the link via normal go to definition.
+
+![scaladoc-def](https://i.imgur.com/TYNMCEm.gif)
+
+To learn more about linking in scaladoc check out the
+[Scala documentation](https://docs.scala-lang.org/scala3/guides/scaladoc/linking.html)
+
+## Custom project root setting
+
+Metals currently will try to find the proper build tool root for your project,
+which is useful especially if you open a parent directory that doesn't contain a
+build tool configuration. However, this can be problematic if you have a less
+traditional setup and you need to be able to control the root directory. For
+example if you have an sbt source dependency and you still want to have that
+dependency open in the same window.
+
+To help with that we added a new setting `metals.customProjectRoot` that allows
+you to specify the root directory for your project. This setting is available in
+the settings UI and can be set to a path relative to the workspace root.
+
+![custom-root](https://i.imgur.com/sgr1j1T.png)
+
+Another amazing feature from [kasiaMarek](https://github.com/kasiaMarek)
+
+## Bsp errors in status
+
+In the previous release Metals started showing more errors from the build server
+with the intention on making it clearer to the user what is going on. However,
+showing a message request for the user could be quite invasive and not always
+relevant.
+
+To help with that [kasiaMarek](https://github.com/kasiaMarek) added a new status
+bar item that will show the number of errors in the build server. This will be
+shown only if there are any errors and will be hidden once the reports connected
```

Comment:
Is there needed for clients to get this new status item? If `bspStatusBarProvider` is `"on"` and implemented then it should _just work_ ™️ right? Or is there any actual changes to the status that need to be account for?